tdp/cli/commands/browse.py

```diff
@@ -162,14 +166,14 @@ def print_formatted_deployment(deployment_log):
 
 def print_formatted_operation_log(operation_log):
     headers = OperationLog.__table__.columns.keys()
-    service_headers = ServiceLog.__table__.columns.keys()
-
+    service_headers = ServiceComponentLog.__table__.columns.keys()
+    # TODO: this outputs Service and ServiceComponent version
```

Comment:
Can you rewrite the `TODO` ? I don't understand what should be done.

Reply:
I added information to the todo